eo-maven-plugin/src/test/java/org/eolang/maven/OptimizeMojoTest.java

```diff
@@ -163,6 +168,60 @@ void testSimpleOptimize(@TempDir final Path temp) throws Exception {
     }
 
     @Test
+    void testSimpleParsingCached(@TempDir final Path temp) throws Exception {
+        final Path src = temp.resolve("foo/x/main.eo");
+        final Path target = temp.resolve("target");
+        new Home().save(
+            "invalid content",
+            src
+        );
+        final Path foreign = temp.resolve("eo-foreign.csv");
+        new FtCached(
+            new HashOfTag("0.28.0").narrow(),
+            target,
+            temp.resolve("optimized")
+        ).save(
+            "foo.x.main",
+            "xmir",
+            () -> new UncheckedText(
+                new TextOf(new ResourceOf("org/eolang/maven/main.xmir"))
+            ).asString()
+        );
+        Catalogs.INSTANCE.make(foreign)
+            .add("foo.x.main")
+            .set(AssembleMojo.ATTR_SCOPE, "compile")
+            .set(AssembleMojo.ATTR_EO, src.toString())
+            .set(AssembleMojo.ATTR_VERSION, "0.28.0");
+        new Moja<>(ParseMojo.class)
+            .with("targetDir", target.toFile())
+            .with("foreign", foreign.toFile())
+            .with("foreignFormat", "csv")
+            .with("cache", temp.resolve("cache"))
+            .execute();
+        new Moja<>(OptimizeMojo.class)
+            .with("targetDir", target.toFile())
+            .with("foreign", foreign.toFile())
+            .with("foreignFormat", "csv")
+            .with("cache", temp.resolve("cache"))
+            .execute();
+        MatcherAssert.assertThat(
+            new Home().exists(
+                target.resolve(
+                    String.format("%s/foo/x/main.%s", OptimizeMojo.DIR, TranspileMojo.EXT)
+                )
+            ),
+            Matchers.is(true)
+        );
+        MatcherAssert.assertThat(
+            new TjSmart(
+                Catalogs.INSTANCE.make(foreign)
+            ).getById("foo.x.main").exists("xmir"),
+            Matchers.is(true)
+        );
+    }
+
+    @Test
+    @Disabled
```

Comment:
@potatmen Why did you disable this test?

Reply:
Fixed

Reply:
@potatmen If so, you need to add a `@todo` comment about it